api/docs/http_catalog.md

```diff
@@ -39,6 +39,8 @@ Four parameters should be provided to the POST or GET:
 doubly-escaped (it should just be singly-escaped).  To keep backward compatibility, the extraneous
 escaping is still used/supported.
 - `transaction_uuid`: a transaction uuid identifying the entire transaction (shows up in the report as well)
+- `static_catalog`: a boolean requesting a static catalog if available; should always be `true`
+- `checksum_type`: a checksum type supported by the agent, for use in file resources of a static catalog. Can be repeated multiple times to list multiple supported types, in order of preference, in case the master doesn't support one or more checksum types.
```

Comment:
Hmm, so munge it manually into a string. We could pick just about any separator, the checksum types are all alphanumeric.